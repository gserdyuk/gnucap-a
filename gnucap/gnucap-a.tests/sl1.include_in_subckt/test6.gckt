* "include" handling
spice
.options parhier=local
.param dummy1=1
* comment 1

.subckt module a b

.param p1=1.e-12
 
v1 a b dc 1
r1 a c 1k
d1 c b dio 

.include 'test6-2.inc'
* comment 2
.echo param in subckt
.param 

.ends


X1 1 0 module

.echo param in top
.param 

.echo circuit list
.list

.print op v(1)
.echo solution
.op

.end
