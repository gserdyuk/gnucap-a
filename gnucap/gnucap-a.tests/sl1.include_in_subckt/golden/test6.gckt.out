Gnucap gnucap-arails  2009.12.07 RCS 26.136 master 2010-04-09
The Gnu Circuit Analysis Package
Never trust any version less than 1.0
Copyright 1982-2009, Albert Davis
Gnucap comes with ABSOLUTELY NO WARRANTY
This is free software, and you are welcome
to redistribute it under the terms of 
the GNU General Public License, version 3 or later.
See the file "COPYING" for details.
param in subckt 
 p1= 1.p p2= 2.
param in top 
 dummy1= 1.
circuit list 
* comment 1 
.subckt module ( a b )
v1 ( a b )  DC  1.
r1 ( a c )  1.K
d1 ( c b )  dio NA( 1.)
.model dio d ( tnom=NA( 27.) is=p1 rs=NA( 0.) n=NA( 1.) tt=NA( 0.) cjo=NA( NA) pb=NA( NA) mj=NA( 0.5) egap=NA( 1.11) xti=NA( 3.) fc=NA( 0.5))
r2 ( a c )  10.K
* comment 2 
.ends module
X1 ( 1 0 )  module 
solution 
#           v(1)      
 27.        1.        
